packages/ra-ui-materialui/src/detail/makeOverridesResolver.tsx

```diff
@@ -0,0 +1,8 @@
+export function makeOverridesResolver(classes: { [key: string]: string }) {
```

Comment:
Would you mind moving this in directly under the `src` directory?

Reply:
Done